src/libelftc_dem_gnu3.c

```diff
@@ -566,7 +567,6 @@ __cxa_demangle_gnu3(const char *org)
 	if (!cpp_demangle_data_init(&ddata, org + 2))
 		return (NULL);
 
-	rtn = NULL;
 	has_ret = more_type = false;
```

Comment:
This also needs to be moved up since it's used in the `clean:` label.

Reply:
Or we just initialize as part of the declaration.

Reply:
If we could use C99 and just declare stuff at the right place, and not use goto's :)